lib/chromedriver.js

```diff
@@ -151,14 +155,30 @@ class Chromedriver extends events.EventEmitter {
     const mapping = await this.getMapping();
     const cds = await this.getChromedrivers(mapping);
 
-    const chromeVersion = semver.coerce(await getChromeVersion(this.adb, this.bundleId));
+    let chromeVersion;
+    if (this.bundleId) {
+      chromeVersion = await getChromeVersion(this.adb, this.bundleId);
+    } else {
+      // we have a webview of some sort, so try to find the bundle version
+      for (const bundleId of WEBVIEW_BUNDLE_IDS) {
```

Comment:
It would still be breaking. Right now the client does not pass in the bundle id, so depending on the order of the list, either for web sessions they would get the version of the chrome-backed webview, or on webview sessions they would get the version of Chrome.

Reply:
But we only use this package in one package, so handling a breaking change is minimally invasive. Especially since that package has to be updated anyway to accommodate it on the calling side.